# Making eDNA FAIR

<div align="center">
    <img src="/assets/images/projectlogo_whiteBG.jpg" alt="logo" width="70%">
</div>

<div style="border: 1px solid black; padding: 10px; font-size: 18px;">
    This is an ongoing, community-led initiative, and we continue to refine and improve the <a href="https://fair-edna.github.io/download.html">products</a> on this site based on feedback and collaboration. Check out the <a href="https://fair-edna.github.io/next.html">Our next steps</a> page to learn more about how we are working with the <a href="https://www.gensc.org/">Genomic Standards Consortium (GSC)</a> and the <a href="https://www.tdwg.org/">Biodiversity Information Standards (TDWG)</a>.
</div>

## Why do we need FAIR eDNA?
The success of environmental DNA (eDNA) approaches for species detection has revolutionised biodiversity monitoring and distribution mapping. Targeted eDNA amplification approaches, such as quantitative PCR, have improved our understanding of species distribution, while metabarcoding-based approaches enable biodiversity assessment at unprecedented scales and taxonomic resolution. eDNA datasets, however, are often scattered across repositories with inconsistent formats, varying access restrictions, and inadequate metadata; this limits their interoperation, reuse, and overall impact. Adopting FAIR (Findable, Accessible, Interoperable, Reusable) data practices with eDNA data can transform monitoring of biodiversity and individual species, and support data-driven biodiversity management across broad scales. FAIR practices remain underdeveloped in the eDNA community, partly due to gaps in adapting existing vocabularies, such as [Darwin Core (DwC)](https://dwc.tdwg.org/) and [Minimum Information about any (x) Sequence (MIxS)](https://genomicsstandardsconsortium.github.io/mixs/), to eDNA-specific needs and workflows. 

## Project scope
Fine-tuning, optimizing and extending existing data and metadata standards specifically for eDNA-based studies is a crucial first step towards achieving FAIR eDNA. To this end, we propose a comprehensive FAIR eDNA (FAIRe) metadata checklist, which integrates existing data standards and introduces new terms tailored to eDNA workflows. 
Metadata are systematically linked to both raw data (e.g., metabarcoding sequences, Ct/Cq values of targeted qPCR assays) and derived biological observations (e.g., ASV/OTU tables, species presence/absence). Along with formatting guidelines, tools, templates and example datasets, we introduce a standardised, ready-to-use approach for FAIR eDNA practices. 
Together, these efforts aim to guide data providers and facilitate the unambiguous description of eDNA datasets that are aligned with FAIR data practices. Our goal is to provide a FAIRe metadata checklist and guidelines as resources for existing data standards, infrastructures and databases, helping them to address gaps to better meet their own objectives. The guidelines will also benefit individual eDNA science practitioners and labs by improving data management standardisation, making their data more FAIR not only for reusers, but also for internal organisation.

<!-- 
<div align="center">
    <embed src="/assets/images/fairedna_infographic.pdf#toolbar=0&navpanes=0" width="600px" height="465px"></embed>
    <p style="margin-top: 0em">Designed by: <a href="https://ooidscientific.com">OOID Scientific</a></p>
</div>
 -->
<div align="center">
    <img src="/assets/images/infographic.jpg" alt="FAIR eDNA infographic" width="90%">
    <figcaption style="margin-top: 0em">Designed by: <a href="https://ooidscientific.com">OOID Scientific</a></figcaption>
</div>



