# Making eDNA FAIR

<div align="center">
    <img src="/assets/images/ednafair_logo.png" alt="logo" width="70%">
</div>

## Project scope

The success of environmental DNA (eDNA) approaches for species detection has revolutionised biodiversity monitoring and distribution mapping. Targeted eDNA amplification approaches, such as quantitative PCR, have improved our understanding of species distribution, while metabarcoding-based approaches enable biodiversity assessment at unprecedented scales and taxonomic resolution. eDNA datasets, however, are often scattered across repositories with inconsistent formats, varying access restrictions, and inadequate metadata; this limits their interoperation, reuse, and overall impact. Adopting FAIR (Findable, Accessible, Interoperable, Reusable) data practices with eDNA data can transform monitoring of biodiversity and individual species, and support data-driven biodiversity management across broad scales. FAIR practices remain underdeveloped in the eDNA community, partly due to gaps in adapting existing vocabularies, such as Darwin Core (DwC) and Minimum Information about any (x) Sequence (MIxS), to eDNA-specific needs and workflows. To address these challenges, we propose a comprehensive FAIR eDNA (FAIRe) Metadata Checklist, which integrates existing data standards and introduces new terms tailored to eDNA workflows. Metadata are systematically linked to both raw data (e.g., metabarcoding sequences, Ct/Cq values of targeted qPCR assays) and derived biological observations (e.g., ASV/OTU tables, species presence/absence). Along with formatting guidelines, tools, templates and example datasets, we introduce a standardised, ready-to-use approach for FAIR eDNA practices. Through broad collaboration, we seek to integrate these guidelines into established biodiversity and molecular data standards, promote journal data policies, and foster user-driven improvements and uptakes of FAIR practices among eDNA data producers. By adopting this standardised approach and developing a long-term plan with key databases and data standard organisations, we aim to enhance accessibility, maximise reuse, and elevate scientific impact of these invaluable biodiversity data resources. 


<!-- 
<div align="center">
    <embed src="/assets/images/fairedna_infographic.pdf#toolbar=0&navpanes=0" width="600px" height="465px"></embed>
    <p style="margin-top: 0em">Designed by: <a href="https://ooidscientific.com">OOID Scientific</a></p>
</div>
 -->
<div align="center">
    <img src="/assets/images/fairedna_infographic.jpg" alt="FAIR eDNA infographic" width="90%">
    <figcaption style="margin-top: 0em">Designed by: <a href="https://ooidscientific.com">OOID Scientific</a></figcaption>
</div>

## Why do we need FAIR eDNA?

The FAIR principles were developed to improve the Findability, Accessibility, Interoperability and Reusability of scholarly digital research objects such as data and scripts (Wilkinson et al., 2016). Alongside open data principles (Mandeville et al., 2021; “Open Data | UNESCO,” 2022; Vicente-Saez and Martinez-Fuentes, 2018), these principles promote the public good and enhance transparency and reproducibility of research. Furthermore, the FAIR principles increase the long-term value of data, generate further knowledge at speed, and serve as a cost-saving measure (Beck et al., 2020; Gomes et al., 2022; Hampton et al., 2013; Jenkins et al., 2023; Kim and Stanton, 2016; Leigh et al., 2024; Sholler et al., 2019; Soranno and Schimel, 2014; Whitlock, 2011). Open and FAIR principles have been adopted worldwide by organisations including Research Data Alliance (RDA, https://www.rd-alliance.org/), Committee on Data of the International Science Council (CODATA, https://codata.org/) and the Future of Research Communications and e-Scholarship (FORCE11) (Jacobsen et al., 2020; Martone, 2015; Wilkinson et al., 2016). FAIR practices, however, have not yet been fully established in the field of environmental DNA (eDNA), limiting the opportunities to unlock its full potential (Berry et al., 2021; Borisenko et al., 2024; Kimble et al., 2022; Shea et al., 2023). 
Given increasing pressures on the environment and accelerated species loss (Ceballos et al., 2015; Diaz et al., 2019), FAIR biodiversity data are particularly needed to support robust, real-time biomonitoring, biodiversity inventories, and to provide a better foundation for  decision-making. Reversing the global loss of biodiversity requires transformational human actions and robust measurements of their effectiveness. Yet, environmental measurement and biodiversity assessment at large scale, with high speed and accuracy, is very challenging. Recently, biodiversity science has adopted eDNA to help address these challenges, noting the various advantages of eDNA including non-invasive sampling, broad taxonomic coverage, high sensitivity, accuracy, scalability and efficiency (Goldberg et al., 2015; Ruppert et al., 2019; Thomsen and Willerslev, 2015). eDNA is also effective in situations where conventional methods for biological observation are impractical or impossible, such as in turbid waters, inaccessible subterranean environments, and for detecting elusive or cryptic species (Hashemzadeh Segherloo et al., 2022; Matthias et al., 2021; Saccò et al., 2022; Villacorta-Rath et al., 2022; West et al., 2020; White et al., 2020). As a result of these advantages, it has been widely implemented for biodiversity detection and monitoring, including national and international strategies, e.g., National Aquatic Environmental DNA Strategy (Kelly et al., 2024), Fisheries and Oceans Canada (Baillie et al., 2019), DNAqua-net (Blancher et al., 2022), the Canadian Standards Association (Helbing and Hobbs, 2019).
There are two primary approaches to detect organisms of interest in eDNA studies. The first approach, targeted assays (e.g., quantitative polymerase chain reaction (qPCR), digital PCR (dPCR)), uses taxon-specific primers and probes to detect the presence of DNA from specific taxa. This method typically targets a single taxon (e.g., species or genus) (but also see Tsuji et al. (2018) and Wilcox et al. (2020) for multiplex and high-throughput PCR) from which eDNA concentration as a proxy for abundance or biomass of focal species may be inferred (Rourke et al., 2022). The second approach, metabarcoding assays, detect entire taxon assemblages  from a sample by amplifying and sequencing genetic marker regions (so-called barcode regions) (Coissac et al., 2012; Hebert et al., 2003). This approach assigns large numbers of DNA sequences to diverse taxa, offering a broad overview of biodiversity within the sample. Both targeted and metabarcoding assay approaches can produce tremendous volumes of complex data. The exact methodologies employed within each approach can vary significantly (Coissac et al., 2012; Takahashi et al., 2023).
Metabarcode sequence variants (amplicon sequence variant (ASV) or operational taxonomic unit (OTU)) are usually taxonomically identified by comparing them to DNA reference databases, such as those available in the Barcode of Life Data System (BOLD) (Ratnasingham and Hebert, 2007) and the International Nucleotide Sequence Database Collaboration (INSDC) (Arita et al., 2021). As reference databases are continuously improved (Delrieu-Trottin et al., 2019; Stoeckle et al., 2020; Weigand et al., 2019), eDNA sequences may receive more precise taxonomic assignment upon reanalysis. The ability to improve in accuracy and precision with time is feature of eDNA data that is rare among biodiversity data types. This potential is, however, dependent on machine readable and high-quality metadata, underscoring the importance of compliance with the FAIR principles. Adhering to these will also expand the lifespan and scale of eDNA data, thereby maximising its power for biodiversity monitoring and providing additional value to existing biodiversity data (Berry et al., 2021; Thompson and Thielen, 2023). For instance, FAIR eDNA data can be reused to increase the temporal and spatial scale of species and community observations (Crandall et al., 2023), fill important spatial and taxonomic gaps in our knowledge of global biodiversity, facilitate early invasive species detection (e.g. Helbing and Hobbs, 2019), and enhance endangered species monitoring. Furthermore, even unassigned sequences can contribute meaningful ecological insights, such as the taxon-independent community index (Wilkinson et al., 2024) or in forensic analyses, habitat classification, and predictive modelling (e.g. Frøslev et al., 2023). Altogether, implementing FAIR principles for eDNA data maximises its research and conservation impact.


