# What's next

## Developing data formatting tools
Data formatting tools, such as R and python scripts accompanied by web user interfaces, are practical for data submitters to format their data according to standard guidelines and correct errors in formats and vocabulary. These tools will also facilitate reformatting from one standard to another. For example, institutional databases may have developed their own data formats and vocabularies. Scripts designed to convert these into the current guidelines will enhance interoperability with eDNA datasets elsewhere and improve readiness for data sharing through open-access platforms. 

GBIF has developed an Integrated Publishing Tool (IPT) to meet such needs and format datasets for submission to GBIF. Similarly, NOAA has developed the software edna2obis to format their datasets to OBIS standards for submission (https://github.com/aomlomics/edna2obis). More tools like these are required to meet the specific needs of individual institutions and databases, ensuring seamless data integration and sharing. 


## Developing taxonomic reannotation scripts
FAIR eDNA datasets will enable us to archive DNA sequences, providing a significant opportunity to periodically re-analyse taxonomic assignments using updated reference databases, thereby improving identification accuracy. This will require the development of standard taxonomic annotation scripts. This can be challenging because researchers develop their scripts using various algorithms, assignment thresholds, and databases, depending on factors such as the characteristics of target gene region, the coverage of reference databases for target taxa, and the stringency requirements for study purposes. Further research is needed to identify and develop a standard pipeline to apply whole eDNA datasets, or alternatively multiple pipelines depending on factors such as target gene region and taxa. This approach will ensure that periodic reanalyses can be conducted to enhance the overall utility and precision of eDNA datasets while allowing customisation to suit specific research needs. 


## Harmonising FAIR and CARE
The CARE principles, which stand for Collective benefit, Authority to control, Responsibility and Ethics, were developed by the Global Indigenous Data Alliance (GIDA) to protect the Indigenous rights and interests in Indigenous data which include information, data, and traditional knowledge about their resources and environments (Carroll et al 2020, Hundson et al 2023). The people- and purpose- oriented CARE principles are designed to complement the data-centric FAIR principles by ensuring that Indigenous data remain FAIR whilst centring Indigenous sovereignty (Cartney et al 2023). Implementing the CARE principles into FAIR data initiatives is essential to ensure that data use respects Indigenous rights, aligns with the openness preferences of Indigenous communities, serves a meaningful purposes, and promotes wellbeing of Indigenous Peoples and Local Communities (IPLC) (Carroll et al 2021). 

Carney et al. (2023) established a roadmap for the lifecycle of eukaryotic biodiversity sequencing data following the CARE principles. It identifies actions required from researchers throughout each study to build sustainable partnerships with IPLC. In the context of FAIR initiatives, it is crucial to develop culturally aware linked metadata. For example, generalising or withholding sensitive data (i.e. geographical coordinates of culturally important sites) is respected and accepted, which can be mapped under the terms dataGeneralisation and informationWithheld in the current eDNA checklist and other standards and databases (Darwin Core Mainenance Group, 2021, Chapman, 2020). Information on access, use permission, and the person or organisation owning and managing data rights, should be recorded under the terms accessRights and rightsHolder. Further works are needed in metadata to document Traditional Knowledge (TK) Labels and Notices (Liggins et al 2021, Local Contexts 2023), provenance about place and people (Carney et al 2023), and the cultural importance of species (Reyes-Gracia et al 2023). Meetings, workshops and collaboration across various stakeholders including IPLC and eDNA communities are essential to recognise the benefits of eDNA data sharing in various aspects, exchange more information and raise discussions. All these efforts are critical to harmonising FAIR and CARE, establishing and retaining the link between people and nature, and serving as a vehicle for knowledge transfer, capacity development, two-way science and biodiversity conservation. 

<div align="center">
<figure>
    <img src="/assets/images/be+fair+care.png" alt="Be FAIR and CARE" width="70%">
    <figcaption style="margin-top: 0em">Image credit: <a href="https://www.gida-global.org/care">https://www.gida-global.org/care</a></figcaption>
</figure>
</div>


## Revising and updating the guidelines
Given eDNA is a relatively new and rapidly evolving field, new methods and technologies will emerge in the near future. Therefore, there is a need to regularly revise and update the current eDNA data checklist and formatting guidelines to ensure that they stay current and continue to support the diverse and growing needs of the eDNA research community. The revision process will involve investigating “other” entries in the fixed-term fields and identifying common entries that may warrant inclusion as fixed-term options. We will also develop additional sample extensions including those from bulk metabarcoding studies, such as planktons, insects and benthic organisms collected using plankton tows, insect traps and benthic substrates, respectively (i.e., Bruce et al. 2017, Alexander et al. 2023, Fernandes et al 2023). We plan to update it annually and welcome feedback from data providers and re-users at any time. 
